EclecticIQ_1.0.0.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Please rename this file to `extension.xml`.

Reply:
done